src/index.html

```diff
@@ -172,7 +172,7 @@ <h1 class="mb-2 font-['Poppins'] text-xl font-medium text-white">SideStore IPA</
             class="flex items-center justify-center rounded-full bg-purple-500 px-7 py-2.5 transition-colors duration-150 ease-in-out hover:bg-purple-600"
           >
             <a
-              href="https://nightly.link/SideStore/SideStore/workflows/build/develop/SideStore.ipa.zip"
+              href="https://github.com/SideStore/SideStore/releases/download/beta/SideStore.ipa"
```

Comment:
We need to add a similar warning as to the one before about the beta/nightly IPAs.

Reply:
Also todo here, change the buttons to the new format.

Reply:
I think we should only have beta on the site, and keep nightly for advanced users/testers. There will be a way to switch in-app or something anyways.

Reply:
Is beta part of the build system?

Reply:
Yes, but it hasn't been triggered. There is also an action for stable builds